core/src/main/java/org/apache/druid/data/input/impl/JsonReader.java

```diff
@@ -106,6 +108,10 @@
       //throw unknown exception
       throw e;
     }
+    if (CollectionUtils.isEmpty(inputRows)) {
```

Comment:
CI is failing asking for 

```<dependency>
  <groupId>commons-collections</groupId>
  <artifactId>commons-collections</artifactId>
  <version>3.2.2</version>
</dependency>
```
to be added to the pom of `druid-core`. Could instead use druid's `org.apache.druid.utils.CollectionUtils.isNullOrEmpty` which does the same check

Reply:
Done